### PR TITLE
Firebase authorization

### PR DESCRIPTION
**Pull request by ihor-mk**
ok
unauthorized

### CODE REVIEW

File: backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -40,5 +43,30 @@ public static void AddEasySpeakCoreContext(this IServiceCollection services, ICo
...
+            var projectId = config.GetValue<string>("project_id");
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                .AddJwtBearer(options =>
+                {
+                    options.Authority = $"https://securetoken.google.com/{projectId}";
+                    options.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidateIssuer = true,
+                        ValidIssuer = $"https://securetoken.google.com/{projectId}",
```
Comment (tatianahutii) [inline backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs:64]: Please remove duplications

File: backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "easy-meets-id-01",
+  "private_key_id": "e62cc972ce15e6979d580a72eac10887b8d5160e",
```
Comment (tatianahutii) [inline backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json:4]: This is not good practice to push such files to the repository

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json (hunk 2 above)
Comment (ihor-mk) [reply backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json]: Only two variables remained

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by bochka123 — DISMISSED** [review]

File: backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "project_id": "easy-meets-id-01",
+  "secure_uri": "https://securetoken.google.com"
+}
```
Comment (tatianahutii) [inline backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json:4]: It would be better to move this to appsettings

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by bochka123 — APPROVED** [review]

**Pull request merged**